firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionModel.swift

```diff
@@ -134,12 +137,15 @@ class TrackingProtectionModel {
 
         let confirmAction = UIAlertAction(title: clearCookiesAlertButton,
                                           style: .destructive) { [weak self] _ in
-            self?.clearCookiesAndSiteData()
+            self?.clearCookiesAndSiteData(cookiesClearable: CookiesClearable(), siteDataClearable: SiteDataClearable())
+            self?.selectedTab?.webView?.reload()
         }
         alert.addAction(confirmAction)
         controller.present(alert, animated: true, completion: nil)
     }
 
-    func clearCookiesAndSiteData() {
+    func clearCookiesAndSiteData(cookiesClearable: Clearable, siteDataClearable: Clearable) {
+        _ = cookiesClearable.clear()
+        _ = siteDataClearable.clear()
```

Comment:
ideally this logic lives elsewhere instead of the model, but I believe you and @petruSt24 will be revisiting the redux architecture in which this may live in the middleware